doc/cdi-populators.md

```diff
@@ -0,0 +1,99 @@
+# CDI Volume Populators
+
+## Recommended knowledge
+
+What are [populators](https://kubernetes.io/blog/2022/05/16/volume-populators-beta/)
+
+## Introduction
+CDI volume populators are CDIs implementation of populating PVCs by importing/uploading/cloning data utilizing the new `dataSourceRef` field. New controllers and custom resources for each population method were introduced.
+
+**Values of the new API?**
+* Native synchronization with Kubernetes - this is kubernetes way of populating PVCs. Once PVC is bound we know it is populated (So far PVC was bound the moment the datavolume created it and the population progress was monitored via the datavolume)
```

Comment:
`Before the PVC was bound the moment the datavolume created it, and the population progress was monitored via the datavolume`

Reply:
`Before introducing populators, the PVC...`